osd_whats_new/osd-whats-new.adoc

```diff
@@ -15,6 +15,20 @@ With its foundation in Kubernetes, {product-title} is a complete {OCP} cluster p
 [id="osd-new-changes-and-updates_{context}"]
 == New changes and updates
 
+[id="osd-q3-2024_{context}"]
+=== Q3 2024
+* **{product-title} regions added.** {product-title} on {GCP} is now available in the following additional regions:
++
+** Melbourne (`australia-southeast2`)
+** Milan (`europe-west8`)
+** Turin (`europe-west12`)
+** Madrid (`europe-southwest1`)
+** Santiago (`southamerica-west1`)
+** Doha (`me-central1`)
+** Dammam (`me-central2`)
++
```

Comment:
Renove the `+` to push the sentence to the edge so that it applies to everything in the list. With the indent, it seems that the link applies only to the last item.
```suggestion

```